### DISCUSSION TITLE
Add copy capability to vector3 values ​​in the transform tree.

### DISCUSSION THREAD

**Discussion by stdgn** [godotengine/godot-proposals#4552] [Editor]
I think it would be nice to have the ability to copy and paste the values ​​of other objects that we want to be the same size or in the same rotation. :)
![41552](https://user-images.githubusercontent.com/39684712/168792096-89b3efa4-29ce-4835-ac86-57490434080b.jpg)

**Comment by Calinou** [comment]
This is already implemented in 4.0alpha and 3.5beta: https://github.com/godotengine/godot/pull/39404

In those versions, this is also accessible via a right-click context menu:

![image](https://user-images.githubusercontent.com/180032/169110923-7363567f-8342-4926-9638-018595a40d06.png)
